### ISSUE TITLE
react 17.0.2 support

### ISSUE THREAD

**Issue by niku2021** [jakezatecky/react-checkbox-tree#395]
I have tried to add this tree view checkbox to my react project. its loading data no issue. but icons are not showing. I am using react 17.0.2. Its working on 16.8.6 version. I am not sure its my issue or this one not supporting. Please add support for new react versions

**Comment by niku2021** [comment]
thanks for your reply. I have created sample project. Can you please check this.

https://codesandbox.io/s/sample-treeview-1zwd3w?file=/src/index.js

**Comment by jakezatecky** [comment]
Thank you for provided the sample project.

It was almost there. It did not include a reference to the Font Awesome CSS. I added this to the index.html:

``` html
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" />
```

One other slight issue is that the Checkbox Tree defaults to the older Font Awesome 4 icons, which are incompatible with Font Awesome 5 or 6. I forgot about that. I had to also set the `iconsClass="fa5"` property:

``` jsx
<CheckboxTree iconsClass="fa5" ... />
```

I will change this in a future release such that it does not default to v4.

See my fork of your CodePen [here](https://codesandbox.io/s/hungry-tdd-nbg66b?file=/public/index.html), which loads all of the icons.
